packages/react-label/src/components/Label/__snapshots__/Label.test.tsx.snap

```diff
@@ -1,9 +1,11 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Label renders a default state 1`] = `
-<div
-  className="__9et6fu0 "
->
-  Default Label
+<div>
+  <div
+    class="__9et6fu0 "
```

Comment:
Can you please add [`jest-serializer-make-styles`](https://github.com/microsoft/fluentui/tree/master/packages/jest-serializer-make-styles) there and into `create-component` template?